src/Kernel32.Shared/Kernel32+OVERLAPPED.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Diagnostics.CodeAnalysis;
+    using System.Runtime.InteropServices;
+
+    /// <content>
+    /// Contains the <see cref="OVERLAPPED"/> nested class.
```

Comment:
Well spotted maybe I should use your phrasing, easier to copy paste (It actually *was* a class but I changed that without changing the comments)